### ISSUE TITLE
Unable to Keep Buttons Pressed

### ISSUE THREAD

**Issue by mariotaku** [moonlight-stream/moonlight-android#994]
**Describe the bug**
While remote playing, holding a controller button (physical or virtual) is supposed to keep it pressing until released. However, while holding the buttons, it behaves like I'm repeatedly pressing it.

**Steps to reproduce**
1. Stream mstsc.exe
2. Open [Gamepad tester](https://gamepad-tester.com/)
3. Press and hold any button (or push joysticks)
4. You can see buttons flashing repeatedly

**Other Moonlight clients**
- Not able to test yet.

**Moonlight settings (please complete the following information)**
- 720p, 60fps, 5mbps
- On screen gamepad on and off
- A pair of Joy-cons connecting via Bluetooth

**Gamepad-related issues (please complete if problem is gamepad-related)**
- No gamepads connected to PC
- Gamepad not correctly functioning with on-screen gamepad on and off
- Problem exists in game and in tester website 

**Device details (please complete the following information)**
 - Android version: 11
 - Device model: Pixel 5

**Server PC details (please complete the following information)**
 - OS: Windows 10 19043.1165
 - GeForce Experience version: 3.23.0.74
 - Nvidia GPU driver: 471.68
 - Antivirus and firewall software: Windows Defender and Windows Firewall

**Additional context**
I'm connecting to the PC via Wireguard VPN to my home network. My phone connects to the VPN server and the server bridges to my home network. It introduces 40ms latency to packets but audio/video works well overall.

**Comment by cgutman** [comment]
I can't reproduce this. My server PC details are the same as yours. I tried on a Galaxy Note 9 running Android 10 and a Pixel 4 running Android 12.

I pushed the A button on the OSC and the button stayed pressed in the gamepad tester while the gamepad timestamp did not advance (indicating no additional gamepad events were being received in the meantime).

It sounds like multiple gamepad inputs (possibly from your Joycons) are all being sent as gamepad 1. Either you have "Automatic gamepad presence detection" turned off, or you have an Accessibility app enabled on your phone and thus are affected by https://issuetracker.google.com/issues/163120692 (where Moonlight's workaround does not allow us to distinguish input from multiple gamepads due to them all coming from the same virtual input device).
